frontend-ssr/pages/submissions.tsx

```diff
@@ -67,6 +94,20 @@ class Submissions extends React.Component<ConnectedProps<typeof connector>, Stat
     }
   };
 
+  loadMore = (entries) => {
+    const { submissions } = this.props;
+
+    entries.forEach((entry) => {
+      if (entry.isIntersecting) {
+        this.getData();
+
+        if (submissions.end) {
```

Comment:
This disconnect code should not be in this function. Instead it should be extracted outside,probably in a `componentDidUpdate`.